src/usr/local/www/services_unbound_host_edit.php

```diff
@@ -225,6 +225,12 @@ function hosts_sort() {
 	));
 }
 
+$section->setHelp("This page is used to override the usual lookup process for a specific host. A host is defined by its name" .
```

Comment:
This doesn't work.
`Fatal error: Call to undefined method Form_Section::setHelp() in /usr/local/www/services_unbound_host_edit.php on line 228 Call Stack: 0.0003 228080 1. {main}() /usr/local/www/services_unbound_host_edit.php:0 PHP ERROR: Type: 1, File: /usr/local/www/services_unbound_host_edit.php, Line: 228, Message: Call to undefined method Form_Section::setHelp()`